### PR TITLE
[docs] Update README.md to include command to install nightly.

### PR DESCRIPTION
**Pull request by ailzhang**
Related issue = #

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,12 +34,27 @@ Advanced features of Taichi include [spatially sparse computing](https://docs.ta
...
 ```
 
 **Supported OS**: Windows, Linux, Mac OS X; **Python**: 3.6-3.9 (64-bit only); **Backends**: x64 CPUs, CUDA, Apple Metal, Vulkan, OpenGL Compute Shaders.
 
+### Nightly releases
+
+Nightly releases with the `master` branch of `taichi` are also available:
+
+```bash
+pip install -i https://test.pypi.org/simple/ taichi-nightly
```
Comment (k-ye) [inline README.md:50]: I remember this could cause a non-stopping installation?

**Review by k-ye — APPROVED** [review]
lgtm!

File: README.md (hunk 1 above)
Comment (ailzhang) [reply README.md:50]: Hmmm it has been working for me. @qiao-bo Do you happen to remember what was the root cause last time it happened on your computer?

Comment (qiao-bo) [reply README.md:50]: It was random, did not happen again.

**Pull request merged**